### DISCUSSION TITLE
Question Regarding TinyMCE React Port to HugeRTE

### DISCUSSION THREAD

**Discussion by vithalreddy** [hugerte/hugerte#40] [Q&A] [answered]
Hi Team,

is there an plan to port [tinymce-react](https://github.com/tinymce/tinymce-react) to hugerte?

and, Thanks for the fork. ik it's an huge effort.

**Comment by carlosmintfan** [comment, marked as answer]
Yes. I‘ll publish the Vue integration today and Svelte, React and Angular integrations will follow. Thank you for your interest!

**Comment by vithalreddy** [comment]
Hi any updates on this? or need any help?

**Reply by carlosmintfan** [reply]
I‘m a slow guy here, sorry, but I hope to publish this week.

**Reply by vithalreddy** [reply]
sure np, please take your time.

**Comment by coderpradp** [comment]
Looking forward to the Angular integration!

**Reply by carlosmintfan** [reply]
I‘ll do Svelte first, but then Angular!

**Reply by carlosmintfan** [reply]
@coderpradp The Angular integration is available since today (in my timezone). #55 

(And Svelte not yet, against my former plan. But it‘ll follow soon. As for myself, Svelte has been my favorite framework, but now I switched to the much simpler [VanJS](https://vanjs.org).)

**Comment by carlosmintfan** [comment]
I have published the React integration on Saturday or Sunday, depending on your timezone, for those who don’t know yet.
